Scripts/script-ADGetGroupMembers.yml

```diff
@@ -10,7 +11,7 @@ script: |+
           endpoint['Type'] = 'AD'
           endpoint['ID'] = demisto.get(l,'dn')
           endpoint['Hostname'] = demisto.get(l,'name')
-          endpoint['Groups'] = demisto.get(l,'memberOf').split('<br>')
+          endpoint['Groups'] = demisto.get(l,'memberOf').split('<br>') if demisto.get(l,'memberOf') else demisto.get(l,'memberOf')
```

Comment:
whats the point of assigning `Groups` fields in `demisto.get(l,'memberOf')` if its falsely?
I recommend:
1. assign `None` to it
OR
2. Don't assign this field at all:
```
if demisto.get(l,'memberOf'):
    endpoint['Groups'] = demisto.get(l,'memberOf').split('<br>')
```

Reply:
Also, the one liner syntax is very hard to understand

Reply:
I think it's readable (this is python ternary operation)
`demisto.get(l,'memberOf').split('<br>') if demisto.get(l,'memberOf') else ''` or `else None`
or
`demisto.get(l,'memberOf') and demisto.get(l,'memberOf').split('<br>')`

Reply:
I know it is python ternary.. but it is still hard to understand.. you can write everything in one liner..